pdfkit/pdfkit.py

```diff
@@ -187,7 +188,13 @@ def to_pdf(self, path=None):
         else:
             input = None
 
-        stdout, stderr = result.communicate(input=input)
+        kwargs = {"input": input}
+
+        # Check if a timeout was specified and pass it along if so
+        if self.timeout:
+            kwargs["timeout"] = self.timeout
+
+        stdout, stderr = result.communicate(**kwargs)
```

Comment:
```suggestion
        stdout, stderr = result.communicate(input=input, timeout=timeout)
```

This should work too?

Reply:
Yep works, can't remember exactly why, I had wanted to avoid passing a `None` value.
However the default value for timeout is `None`, so definitely makes sense to just do this.